### DISCUSSION TITLE
Change type of tooltip for Text Card mod

### DISCUSSION THREAD

**Discussion by afaskowitz** [TIBCOSoftware/spotfire-mods#114] [Q&A] [answered]
Hi, I am wondering how easy is it to get the tooltip of the text card mod working like other visualizations? The tooltip on the text card is a link that I want to render as an image through the url, but the tooltip of the mod seems to be set up differently where I can't set a different rendering method. The other mod in my screenshot (the word cloud) looks to have the functionality I'm looking for. Could anyone point me in the right direction here? Thanks.

![screenshot_2023-05-01_at_2 57 43_pm](https://user-images.githubusercontent.com/78978576/235548457-8943c8b4-f2f7-4959-9561-984bbeaefef1.png)

**Comment by objerke** [comment, marked as answer]
Hi @afaskowitz,
In order to get Spotfire native tooltips in the Text card mod you need to update to version 1.3 of the Mods API and use the new tooltip functionality.

This is done by:
- Updating the [apiVersion to 1.3](https://github.com/TIBCOSoftware/spotfire-mods/blob/a98d014abaf7ea514a711b0747eb97dbb99fe731/catalog/configurable-gauge/static/mod-manifest.json#L2) in the mod-manifest.json
- Adding the [`"tooltip": {"enabled": true}`](https://github.com/TIBCOSoftware/spotfire-mods/blob/a98d014abaf7ea514a711b0747eb97dbb99fe731/catalog/configurable-gauge/static/mod-manifest.json#L40) to the `dataViewDefinition` property in the mod-manifest. 
- Use the [Tooltip.show()](https://tibcosoftware.github.io/spotfire-mods/api-docs/interfaces/Tooltip.html#show) with a DataViewRow as [argument](https://github.com/TIBCOSoftware/spotfire-mods/blob/a98d014abaf7ea514a711b0747eb97dbb99fe731/catalog/configurable-gauge/src/index.ts#L113). Then the native tooltip will be shown.

I hope this will help you.

**Reply by afaskowitz** [reply]
Thanks for the help, was able to get it working
